src/lib/io/export.ts

```
@@ -0,0 +1,143 @@
+
+  if (structure.id) parts.push(structure.id) // Add ID if available
+
+  // Add formula
+  const formula = electro_neg_formula(structure)
+  if (formula && formula !== `Unknown`) parts.push(formula)
+
+  // Add space group if available
+  if (
+    `symmetry` in structure &&
+    structure.symmetry &&
+    typeof structure.symmetry === `object` &&
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)